screen2.0/src/app/search/types.ts

```diff
@@ -190,4 +190,6 @@ export type URLParams = {
     MammalEnd: number
     VertebrateStart: number
     VertebrateEnd: number
+    Accessions: string
```

Comment:
variable name case can be changed to to camelCase to keep it consistent across the code